### PR TITLE
Added my name, Check it.

### PR DESCRIPTION
**Pull request by Maverick-99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 |   Name                         | DSA  |  GSOC  |
 | ---                            | ---  | ---    |
 | Kushagra Shekhawat             |YES   |YES     |
+| Janmaijai rana                 |YES  |YES    |
```
Comment (kushagra414) [inline README.md]: Correct spacing of "|" after YES

File: README.md [hunk 2]
```diff
@@ -52,4 +53,4 @@
 #### Tasks
 - [ ] Learn Git/GitHub.
 - [ ]  Submit your Name for 100 Days of Code.
-> **IDIOT BOX** : Empty
\ No newline at end of file
+> **IDIOT BOX** : Empty
```
Comment (kushagra414) [inline README.md]: Why did you change this line?
Also, you have to add a folder with your name.
Please read the README file again.

**Comment by kushagra414** [comment]
#1

**Comment by Maverick-99** [comment]
check  8964c3d1ceafb3052d3f2694d67b91d7585c19f5

**Comment by Maverick-99** [comment]
NEWLY UPDATED 8964c3d1ceafb3052d3f2694d67b91d7585c19f5

**Comment by Maverick-99** [comment]
check #3
